src/js/components/Alert.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-const Alert = ({ type = 'info', heading, children }) => {
+const Alert = ({ type = 'info', heading, linebreak, children }) => {
```

Comment:
Same as the last PR, I don't think we need the break (`<hr />`) here. The content is good without it.

Reply:
Yeah, this was actually just a reorg of code from that first PR anyhow, knew there was going to be more changes here once that was merged ^^